app/test_main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+from app.main import can_access_google_page
+from unittest import mock
+import pytest
+
+
+@mock.patch("app.main.valid_google_url")
+@mock.patch("app.main.has_internet_connection")
+@pytest.mark.parametrize(
+    "validate_google_url, has_internet_connection, expected_result",
+    [
+        (True, True, "Accessible"),
+        (True, False, "Not accessible"),
+        (False, True, "Not accessible"),
+        (False, False, "Not accessible"),
```

Comment:
Add an error message. It's good when the user understands what the problem is. Fix everywhere